### ISSUE TITLE
Agricraft 1.4.0 broke some custom crops

### ISSUE THREAD

**Issue by jikuja** [jadedcat/AS2Configs#204]
https://github.com/InfinityRaider/AgriCraft/commit/8ff15b09c2e603818d6e6367ab284b5c9697ca92 makes agricraft to ingore some custom crops. Extra spaces before cusom crop information should be removed from config file.

Example from log:
```
[02:44:04] [Client thread/INFO] [AgriCraft/AgriCraft]: Error when adding custom crop: Incorrect amount of arguments, arguments should be: (name, fruit:fruitMeta, soil, baseBlock:baseBlockMeta, tier, renderType, information, shearable (optional) ) (line: ghastlymoss,minecraft:ghast_tear,minecraft:soul_sand,AWWayofTime:AlchemicalWizardrybloodRune:0,3,1, Much quieter than hunting)
[02:44:04] [Client thread/INFO] [AgriCraft/AgriCraft]: Error when adding custom crop: Incorrect amount of arguments, arguments should be: (name, fruit:fruitMeta, soil, baseBlock:baseBlockMeta, tier, renderType, information, shearable (optional) ) (line: arditeclematis,TConstruct:materials:29,minecraft:soul_sand,exastris:ArditeOreBlock:2,2,1, The nether is no fun)
[02:44:04] [Client thread/INFO] [AgriCraft/AgriCraft]: Error when adding custom crop: Incorrect amount of arguments, arguments should be: (name, fruit:fruitMeta, soil, baseBlock:baseBlockMeta, tier, renderType, information, shearable (optional) ) (line: arditebush,TConstruct:materials:4,minecraft:soul_sand,exastris:ArditeOreBlock:2,3,1, The nether is no fun)
[02:44:04] [Client thread/INFO] [AgriCraft/AgriCraft]: Error when adding custom crop: Incorrect amount of arguments, arguments should be: (name, fruit:fruitMeta, soil, baseBlock:baseBlockMeta, tier, renderType, information, shearable (optional) ) (line: cobaltclematis,TConstruct:materials:28,minecraft:soul_sand,exastris:CobaltOreBlock:2,2,1, The nether is no fun)
[02:44:04] [Client thread/INFO] [AgriCraft/AgriCraft]: Error when adding custom crop: Incorrect amount of arguments, arguments should be: (name, fruit:fruitMeta, soil, baseBlock:baseBlockMeta, tier, renderType, information, shearable (optional) ) (line: cobaltbush,TConstruct:materials:3,minecraft:soul_sand,exastris:CobaltOreBlock:2,3,1, The nether is no fun)
```

**Comment by MrSpider** [comment]
Can confirm bug. Removing the spaces in front of the last argument fixes things.

**Comment by cblair91** [comment]
This was already fixed by 1e7ca58
